yarn.lock

```diff
@@ -7293,7 +7309,7 @@ bn.js@>4.0.0, bn.js@^5.1.1, bn.js@^5.1.2, bn.js@^5.1.3, bn.js@^5.2.0, bn.js@^5.2
   resolved "https://registry.yarnpkg.com/bn.js/-/bn.js-5.2.1.tgz#0bc527a6a0d18d0aa8d5b0538ce4a77dccfa7b70"
   integrity sha512-eXRvHzWyYPBuB4NBy0cmYQjGitUrtqwbvlzP3G6VFnNRbsZQIxQ10PbKKHt8gZ/HW/D/747aDl+QkDqg3KQLMQ==
 
-bn.js@^4.0.0, bn.js@^4.1.0, bn.js@^4.1.1, bn.js@^4.11.0, bn.js@^4.11.6, bn.js@^4.11.7, bn.js@^4.11.8, bn.js@^4.11.9:
+bn.js@^4.0.0, bn.js@^4.1.0, bn.js@^4.1.1, bn.js@^4.11.0, bn.js@^4.11.6, bn.js@^4.11.7, bn.js@^4.11.8, bn.js@^4.11.9, bn.js@^4.4.0:
```

Comment:
@kumavis has a PR open to deduplicate a bunch of these #16410 -- Could you see if you can pin your package to use a version that is corresponding to that PR?

Reply:
I'll dig into this Brad and see if we can't get it aligned

Reply:
Hey Brad! 

We tried a couple things to pin the dependency but it looks like the node v16 (dictating the npm version) in the .nvmrc as well as the pinned yarn v1.6 in the package.json engines flag are too outdated to appropriately do this. 

Regarding the npm version: It looks like node v16 is using npm v8.17. The NPM overrides call dictated [here](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) that can be used to pin nested dependencies is only supported on v8.30 and up. 

Regarding the yarn version: We tried doing different variations of this
```
"resolutions": {
    ...
    "@unstoppabledomains/resolution/**/bn.js": "^4.11.6"
  },
```
But because the unstoppable package name includes a '/' (@unstoppabledomains/resolution) is seems there is a bug in yarn that won't recognize the pin. It was fixed in yarn 2.0 and wasn't backported. related yarn issue [here](https://github.com/yarnpkg/yarn/issues/4874)

Very open to any ideas you might have to pin as the devDependency for the resolution library uses bn.js v4.11.6 which is referenced in #16410 so if we can find a way to pin the dependency to that we should be OK.